### DISCUSSION TITLE
Installation Issue (VirtualBox 6.1 using Security Onion 2.3.21)

### DISCUSSION THREAD

**Discussion by securepadawan** [Security-Onion-Solutions/securityonion#2742] [Unsupported Versions] [answered]
I have tried installing Security Onion on Virtual box on both a Windows 10 desktop and Linux (Pop OS) laptop and keep getting a never-ending reboot cycle. I never get to the Setup screen after the initial reboot. After the first reboot, it takes me back to the "Installing the Security Onion ISO on this device will DESTROY ALL DATA and partitions!" screen. Once I type yes, it wants me to create the admin username again. On the same desktop, I was able to get it installed without any issue on VMWare, however, my license is about to expire. I wanted to move over to VirtualBox

I did allocate 16 GB RAM, 200 GB hard disk, 5 Processor cores, as well as configured 2 NICs (NAT and Host-only adapter). This was configured based on the instructions located here: https://docs.securityonion.net/en/2.3/virtualbox.html
I have no issues installing an older version of Security Onion (16.04.7). Do you have any suggestions on what else I need to go to get it working?

Thanks

**Comment by TOoSmOotH** [comment, marked as answer]
Remove the CD drive from the VM

**Reply by securepadawan** [reply]
Awesome! Thanks for this. I recreated my VM in VirtualBox and let it reboot. Once I got the installation message, I shut down my VM and went in and removed my Floppy and Optical drive from the Boot Order (Setting-> System and just left Hard Drive checked). I booted it back up and got the setup screen and now able to continue.
